databases/week2/studentdb.sql

```diff
@@ -0,0 +1,25 @@
+CREATE DATABASE schoolstudent_db;
+USE schoolstudent_db;
+-- class table content 
+CREATE TABLE class(
+id int unsigned primary key auto_increment,
+classname varchar(255) not null,
+begins datetime,
+ends datetime
+);
+-- student table content
+CREATE TABLE student(
+id int unsigned primary key auto_increment,
+studentname varchar(255) not null,
+email varchar(255) not null unique,
+phone int unique,
+class_id int unsigned not null,
```

Comment:
The `class_id `is not the attribute of a student class.
A students has many classes and a class has many students.
You need to add another table `class_students `to save the relation because a student has many classes and a class has many students.

Reply:
How is this resolved?